pkg/maps/tunnel/tunnel.go

```diff
@@ -42,7 +42,9 @@ func SetTunnelMap(m *Map) {
 
 func TunnelMap() *Map {
 	tunnelMapInit.Do(func() {
-		tunnelMap = NewTunnelMap(MapName)
+		if tunnelMap != nil {
```

Comment:
Shouldn't this be `if tunnelMap == nil`?

Reply:
:facepalm: yes thank you.